ElectronicObserver/Window/Wpf/ExpeditionCheck/ExpeditionCheckViewModel.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using ElectronicObserver.Behaviors.PersistentColumns;
+using ElectronicObserver.Data;
+using ElectronicObserver.Observer;
+using ElectronicObserver.Resource;
+using ElectronicObserver.ViewModels;
+using ElectronicObserver.ViewModels.Translations;
+
+namespace ElectronicObserver.Window.Wpf.ExpeditionCheck;
+
+public class ExpeditionCheckViewModel : AnchorableViewModel
+{
+	public ExpeditionCheckTranslationViewModel ExpeditionCheckTranslation { get; }
+
+	public ObservableCollection<ExpeditionCheckRow> Rows { get; } = new();
+
+	public List<ColumnProperties> ColumnProperties { get; set; } = new();
+	public List<SortDescription> SortDescriptions { get; set; } = new();
+
+	public ExpeditionCheckViewModel() : base(Ioc.Default.GetService<ExpeditionCheckTranslationViewModel>()!.Title, "ExpeditionCheck",
+		ImageSourceIcons.GetIcon(IconContent.FormExpeditionCheck))
+	{
+		ExpeditionCheckTranslation = Ioc.Default.GetService<ExpeditionCheckTranslationViewModel>()!;
+
```

Comment:
Currently, the title won't get updated when the language gets changed. You need to subscribe to the translations viewmodel property changed and update the title.

```cs
ExpeditionCheckTranslation.PropertyChanged += (_, _) => Title = ExpeditionCheckTranslation.Title;
```